algorithms-and-data-structures/basic-javascript/index080.js

```diff
@@ -0,0 +1,9 @@
+let xp = 0;
+let health = 100;
+let gold = 50;
+let currentWeapon = 0;
+let fighting;
+let monsterHealth;
+let inventory = "stick";
+
+// Since the inventory can store multiple items, change the value of inventory to an array with the items stick, dagger, and sword. Here is an example of a variable sandwich that equals a three-item array: let sandwich = ["peanut butter", "jelly", "bread"]
```

Comment:
```suggestion
/*
Since the inventory can store multiple items, change the value of inventory to an array with the items stick, dagger, and sword. 
Here is an example of a variable `sandwich` that equals a three-item array: `let sandwich = ["peanut butter", "jelly", "bread"]`
*/
```